geo/src/algorithm/affine_ops.rs

```diff
@@ -115,6 +115,19 @@ impl<T: CoordNum, M: MapCoordsInPlace<T> + MapCoords<T, T, Output = Self>> Affin
 ///     (x: -0.5688687, y: 5.5688687)
 /// ], max_relative = 1.0);
 /// ```
+///
+/// ## Create affine transform manually, and access elements using getter methods
+/// ```
+/// let transform = AffineTransform::new(10.0, 0.0, 400_000.0, 0.0, -10.0, 500_000.0);
+///
+/// let a: f64 = transform.a();
+/// let b: f64 = transform.b();
+/// let c: f64 = transform.xoff();
+/// let d: f64 = transform.d();
+/// let e: f64 = transform.e();
+/// let f: f64 = transform.yoff();
```

Comment:
To make this an actual doctest, you could create a second transform with these values and assert it's equal?

```rs
assert_eq!(transform, AffineTransform::new(a, b, c, d, e, f))
```

Reply:
Ok, done at 771061c96a5a0ac6314e129db67d5e3907867f09!